### DISCUSSION TITLE
Volvo EX30

### DISCUSSION THREAD

**Discussion by pvhpsb** [meatpiHQ/wican-fw#158] [Q&A]
I use the Wican with Home Assistant to use the SoC, which works fine.
Now I want to use the Wican with Car Scanner App.
Connect to ELM327 is OK, but no connection with the car.
Does anyone have it working with the EX30? if yes please share the Car Scanner settings.

**Comment by meatpiHQ** [comment]
@pvhpsb have you tried setting up the ECU protocol manually to 11bit 500 Kbaud? You can this setting in adapter advanced settings.

**Comment by pvhpsb** [comment]
That was the solution, thanks

**Comment by treynaer** [comment]
Hi,
Can you share your config?
What PIDs are you using?
Thanks

**Reply by meatpiHQ** [reply]
@treynaer @pvhpsb If you're able to share some car scanner logs and PIDs I can create a new vehicle profile for the EX30, so you can use it with the new HA integration.

https://github.com/meatpiHQ/wican-fw/tree/main/vehicle_profiles

**Reply by treynaer** [reply]
I'm using this for now: https://www.swedespeed.com/threads/ex30-soc-in-home-assistant.678941/
I'm available to test a vehicle profile otherwise.

**Reply by meatpiHQ** [reply]
@treynaer can you please follow the instructions in the link to export the SoC logs? Make sure to note the SoC.

https://meatpihq.github.io/wican-fw/car_scanner/logs
